pubsub/pubsub.go

```diff
@@ -22,16 +22,30 @@ import (
 
 // PubSub is the interface for message buses.
 type PubSub interface {
+	BulkMessager
 	Init(metadata Metadata) error
 	Features() []Feature
 	Publish(req *PublishRequest) error
 	Subscribe(ctx context.Context, req SubscribeRequest, handler Handler) error
 	Close() error
 }
 
+// BulkMessager is the interface defining BulkPublish and BulkSubscribe definitions for message buses
```

Comment:
```suggestion
// BulkMessager is implemented by any pubsub component that has BulkPublish and BulkSubscribe methods. 
```

similar to https://pkg.go.dev/fmt#Stringer